engine/src/main/java/com/datatorrent/stram/util/SecurityUtils.java

```diff
@@ -87,9 +105,59 @@ public static boolean isHadoopWebSecurityEnabled()
     return hadoopWebSecurityEnabled;
   }
 
-  public static boolean isStramWebSecurityEnabled()
+  @VisibleForTesting
+  protected static boolean isStramWebSecurityEnabled()
   {
     return stramWebSecurityEnabled;
   }
 
+  /**
+   * Setup security related configuration for {@link org.apache.hadoop.yarn.webapp.WebApp}.
+   * @param config
+   * @param sslConfig
+   * @return
+   */
+  public static Configuration configureWebAppSecurity(Configuration config, SSLConfig sslConfig)
```

Comment:
Build sounds like a good idea. I am also ok with any of these names. Let me know.